pelican/plugins/myst_reader/myst_reader.py

```diff
@@ -83,7 +83,7 @@ def _create_html(self, source_path, content):
         """Create HTML5 content."""
 
         # Find and add bibliography if citations are specified
-        if "{cite" in content:
+        if "{cite}" in content:
```

Comment:
Oh I think it was better with only `"{cite"`. See the doc of the extension.

Reply:
Aha I assumed it was a typo. Now I get it.  I missed this since there was no mention of it in myst-parser docs. Had to find it from the sphinxext and JS library docs

- https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html
- https://sphinxcontrib-bibtex.readthedocs.io/en/latest/usage.html#markdown-syntax-using-myst
- https://js.myst.tools/guide/citations